test/greenbar/tags/attachment_test.exs

```diff
@@ -0,0 +1,124 @@
+defmodule Greenbar.Tags.AttachmentTest do
+  use Greenbar.Test.Support.TestCase
+  alias Greenbar.Engine
+
+  setup_all do
+    {:ok, engine} = Engine.new
+    [engine: engine]
+  end
+
+  defp eval_template(engine, name, template, args) do
+    engine = Engine.compile!(engine, name, template)
+    Engine.eval!(engine, name, args)
+  end
+
+  test "attachment with body", context do
+    result = eval_template(context.engine,
+                           "basic_attachment",
+                           "~attachment~body~end~",
+                           %{})
+
+    assert [%{name: :attachment,
+              fields: [],
+              children: [
+                %{name: :text, text: "body"},
+                %{name: :newline}
```

Comment:
Not entirely sure. Need to spend more time digging into where that's getting injected.